Bump @nextcloud/vue to v8-beta

### Is your feature request related to a problem? Please describe.

While upgrading https://github.com/nextcloud/server to `@nextcloud/vue@8` the following error is thrown
```bash
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @nextcloud/calendar-availability-vue@0.6.0-alpha1
npm ERR! Found: @nextcloud/vue@8.0.0-beta.1
npm ERR! node_modules/@nextcloud/vue
npm ERR!   @nextcloud/vue@"^8.0.0-beta.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @nextcloud/vue@"^6.0 || ^7.0" from @nextcloud/calendar-availability-vue@0.6.0-alpha1
npm ERR! node_modules/@nextcloud/calendar-availability-vue
npm ERR!   @nextcloud/calendar-availability-vue@"^0.6.0-alpha1" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @nextcloud/vue@7.12.0
npm ERR! node_modules/@nextcloud/vue
npm ERR!   peer @nextcloud/vue@"^6.0 || ^7.0" from @nextcloud/calendar-availability-vue@0.6.0-alpha1
npm ERR!   node_modules/@nextcloud/calendar-availability-vue
npm ERR!     @nextcloud/calendar-availability-vue@"^0.6.0-alpha1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Describe the solution you'd like

Bump @nextcloud/vue to v8-beta and pre-release

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "@nextcloud/browserslist-config": "^2.2.0",
     "@nextcloud/eslint-config": "^8.2.0",
     "@nextcloud/l10n": "^1.4 || ^2.0",
-    "@nextcloud/vue": "^6.0 || ^7.0",
+    "@nextcloud/vue": "^8.0.0-beta.1",
     "@vitejs/plugin-vue2": "^2.2.0",
     "browserslist-to-esbuild": "^1.2.0",
     "sass": "^1.57.1",
@@ -50,7 +50,7 @@
   },
   "peerDependencies": {
     "@nextcloud/l10n": "^1.4 || ^2.0",
-    "@nextcloud/vue": "^6.0 || ^7.0",
+    "@nextcloud/vue": ">=8.0.0-beta.0",
     "vue": "^2.7"
   },
   "browserslist": [

